opendcs-rest-api-jetty/Dockerfile-install-postgres-schema

```diff
@@ -6,6 +6,7 @@ RUN apt-get install openjdk-17-jre -y
 COPY ./docker-config/ /opendcs/
 COPY ./build/tmp/opendcs-installer.jar /opendcs/opendcs-installer.jar
```

Comment:
fyi, 7.0.12 has docker images, you could just derive from it and avoid the install.

Reply:
if you are willing to risk 7.0.13 there is a "migateDatabase" app that will do the schema install, including baseline data.

Reply:
i just looked at the first line. don't do this. postgres should be it's own service in the compose file. 

mixing things in this way is going to be super unstable.

Reply:
Yeah the plan is to throw this all away and update to the migrateDatabase app